Jenkinsfile

```diff
@@ -13,33 +13,16 @@ def getLastNonAbortedBuild(build) {
   }
 }
 
-// Send out a slack message if branch got broken or has recovered
```

Comment:
We need to replace alerting. If we are not doing that with this PR , then I will suggest to please create a task for this so that we don't miss it.

Reply:
There's no alerting today as we don't have our slack subscription anymore, I'll mention this in the standup.